contrib/kafka/pkg/utils/util.go

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
this file was moved. I only did cosmetic changes to remove the `provisioner` word.